lib/facilities/vha_facility.rb

```diff
@@ -115,23 +103,10 @@ def outpatient_surgical_specialty_list
       end
 
       def mapped_fields_list
-        %w[StationNumber StationName CocClassification FacilityDataDate Website_URL Latitude
-           Longitude Street Building Suite City State Zip Zip4 MainPhone MainFax AfterHoursPhone
-           PatientAdvocatePhone EnrollmentCoordinatorPhone PharmacyPhone MHPhone Extension Monday
-           Tuesday Wednesday Thursday Friday Saturday Sunday SHEP_Primary_Care_Routine
-           SHEP_Primary_Care_Urgent Hematology SHEP_Specialty_Care_Routine
-           SHEP_Specialty_Care_Urgent SHEP_ScoreDateRange PrimaryCare MentalHealthCare
-           DentalServices Audiology ENT
-           ComplementaryAlternativeMed DiagnosticServices ImagingAndRadiology LabServices
-           EmergencyDept EyeCare OutpatientMHCare OutpatientSpecMHCare VocationalAssistance
-           OutpatientMedicalSpecialty AllergyAndImmunology CardiologyCareServices UrgentCare
-           DermatologyCareServices Diabetes Dialysis Endocrinology Gastroenterology
-           InfectiousDisease InternalMedicine Nephrology Neurology Oncology
-           PulmonaryRespiratoryDisease Rheumatology SleepMedicine OutpatientSurgicalSpecialty
-           CardiacSurgery ColoRectalSurgery
-           GeneralSurgery Gynecology Neurosurgery Orthopedics PainManagement
-           PlasticSurgery Podiatry
-           ThoracicSurgery Urology VascularSurgery Rehabilitation WellnessAndPreventativeCare]
```

Comment:
I didn't realize how many unused fields we had mapped for validation purposes.

Reply:
The validation also only subtracts one way. If we include a field in this mapping, we fail the validation if it doesn't exist in the object that we receive. But we don't fail if the object starts adding additional fields.